### ISSUE TITLE
DualShock 4 Wireless Adaptor HID Driver

### ISSUE THREAD

**Issue by reztekado** [libretro/RetroArch#4406]
Since the DualShock 4 Wireless Adaptor output is pretty much the same as the controller native USB port, is possible to add a HID driver for it?

**Comment by fr500** [comment]
why? it works just fine as it is now.
for OSX?

**Comment by reztekado** [comment]
No, Wii (only HID interface available)

**Comment by twinaphex** [comment]
Yes, the low-level HID Implementation doesn't currently doesn't work on the Wii, it does work on Mac and PC.

If you look at the PS3 HID driver implementation, there were some ifdefs made there to make it work for Wii. Something similar would have to be done for the PS4 driver.
